test/controllers/root-controller.test.js

```diff
@@ -351,6 +352,7 @@ describe('RootController', function() {
       resolveClone();
 
       assert.isTrue(cloneRepositoryForProjectPath.calledWith('git@github.com:atom/github.git', '/home/me/github'));
+      assert.isTrue(reporterProxy.addEvent.calledWith('clone-repo', {package: 'github'}))
```

Comment:
Looks like `addEvent` gets called asynchronously after the `cloneRepositoryForProjectPath` promise is resolved. We can account for this using our async assertion (courtesy of @BinaryMuse ✨):

```
      await assert.async.isTrue(reporterProxy.addEvent.calledWith('clone-repo', {package: 'github'}));
```